docs/monitoring/rsv.md

```diff
@@ -0,0 +1,334 @@
+
+    !!! note
+        The `update` command will update **all** packages on your system.
+
+2. If you have installed HTCondor already but not by RPM, install a special empty RPM to make RSV happy:
+
+        :::console   
+        [root@client ~] # yum install empty-condor --enablerepo=osg-empty
+
+3. Install RSV and related software: 
+
+        :::console
+        [root@client ~] # yum install rsv  
+   
+Configuring RSV
+---------------
+ 
+After installation, there are some one-time configuration steps to tell RSV how to operate at your site.
+ 
+1. Edit `/etc/osg/config.d/30-rsv.ini` and follow the instructions in the file. There are detailed comments for each setting. In the simplest case — to monitor only your CE — set the `htcondor_ce_hosts` variable (or `gram_ce_hosts` for a GRAM CE) to the fully qualified hostname of your CE. For a sample `rsv.ini` file, see the complete installation output [below](#InstallDetails)
+
+2. If you have installed HTCondor already but not by RPM, specify the location of the Condor installation in `30-rsv.ini` in the `condor_location` setting. If an HTCondor RPM is installed, you do not need to set `condor_location`.
+
+3. Complete the configuration using the `osg-configure` tool:
+
+        :::console
+        [root@client ~] # osg-configure -v
+        [root@client ~] # osg-configure -c
+
+    !!! note
+        The `osg-configure` tool produces a lot of output; see [below](#OsgConfigDetails) for an example.
+
+### Optional configuration
+
+The following configuration steps are optional and will likely not be required for setting up a small or typical site. If you do not need any of the following special configurations, skip to [the section on using RSV](#using-rsv).
+
+Generally speaking, read the ConfigureRsv page for more advanced configuration options. Or see [below](#ConfigFileDetails) for specific advanced configuration scenarios.
+
+#### Configuring RSV to run probes using a remote server
+
+RSV monitors systems by running probes, which can run on the RSV host itself (the default case), via a separate batch system like HTCondor, or via a remote batch system using a Globus gatekeeper and its job manager. The last two options both can count those jobs and report them to, for example, Gratia.
+
+In this case, remember to:
+
+- Add the RSV user `rsv` on all the systems where the probes may run, and
+- Map the RSV service certificate to the user you intend to use for RSV. This should be a local user used exclusively for RSV and not belonging to an institutional VO to avoid for the RSV probes to be accounted as regular VO jobs in Gratia. This can be done in [GUMS](../security/gums.md) or [using a grid-mapfile-local](../security/Edg-mkgridmap.md) (if you use a grid-mapfile). MapServiceCertToRsvUser explains how to configure GUMS or the grid-mapfile. Also see the [CE installation document](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/NavAdminCompute) for more information.
```

Comment:
The 'GUMS' and 'using a grid-mapfile local' links are broken because those docs haven't been converted yet. Change the links to point to the twiki; we'll fix them once those pages get converted.